ROMFS/px4fmu_common/init.d/rc.mc_apps

```diff
@@ -8,7 +8,7 @@
 # INAV, which defaults to 0 now. 
 if param compare INAV_ENABLED 1
 then
-	attitude_estimator_ekf start
+	attitude_estimator_q start
```

Comment:
I'm not sure that running the control loop faster is a good idea. I would love to see the impact on current / flight time. The props turn only at 120 Hz, so updating the motor speed 4 times a revolution doesn't make any difference, but it likely will burn more energy. But validating that analysis with a test and seeing how it affects current / performance would be good.